pkg/cloud/services/network/subnets_test.go

```diff
@@ -2067,6 +2090,168 @@ func TestDeleteSubnets(t *testing.T) {
 			},
 			errorExpected: false,
 		},
+		{
+			name: "unmanaged vpc, remove tags - success",
```

Comment:
Can we follow naming convention for the new tests being added? Please refer [here](https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/3198).